README.md

```diff
@@ -1 +1,185 @@
+- An IDE (e.g.: Vscode, Sublime Text,...).
+- A terminal (Git Bash).
+- Install and use Webpack project bundler.
+
+### Setup
+
+Clone this repository to your desired folder by running the following commands in your terminal:
+
+```sh
+  cd your-prefered-folder-name
+  
+  git clone git@github.com:78fatemanazari/To-Do-List.git
+```
+
+### Install
+
+- Install this project with linters and Webpack bundler:
+
+```sh
+  cd to_do_list
+
+  npm init -y
+
+  npm install --save-dev hint@7.x
+
+  npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+  npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+
+  install & configure Webpack bundler with npm, lodash, an HTMLWebpack plugin
+  and a Webpack-dev-server
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  Open the project in Webpack dev server or Open the index.html file, from dist/ folder, on your browser.
+```
+### Build the project with Webpack and run a dev server
+```
+  Run the following commands in the terminal:
+
+  npm run build
+  
+  npm start
+```
+
+### Run Linters tests
+
+To run tests, run the following command:
+
+```sh
+  npx hint . --fix
+  
+  npx stylelint "**/*.{css,scss}" --fix
+
+  npx eslint . --fix
+```
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Juan Carlos Muñoz**
+
+- GitHub: [@78fatemanazari](https://github.com/78fatemanazari)
+- Twitter: [@fati_nazari78](https://twitter.com/fati_nazari78?s=31)
+- LinkedIn: [78fatemanazari](https://www.linkedin.com/in/78fatemanazari)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+The To Do List website will offer the following features:
+- **Data storage in the browser's local storage**
+- **CRUD task operations (Create, Read, Update, Delete)**
+- **Updating & removing completed tasks**
+- **Reorder data array after deleting tasks**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+All contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐ Show your support <a name="support"></a>
+
+If you like this project, please support me with a like or a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Thanks to my learning partners.
+- Thanks to the Microverse team for the great curriculum.
+- Thanks to the Code Reviewer(s) for the insightful feedbacks.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
+
```

Comment:
- [ ] The link to your LICENSE on the README displays a 404 error page when clicked. The path defined is incorrect and so, it won't display the LICENSE for this project. Kindly use the correct path 👍 😎 👇:

```md
This project is [MIT](./LICENSE) licensed.
```